### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by mcsteel88** [Serene-Arc/bulk-downloader-for-reddit#555] [General]
I am getting the below error when trying to run the setup file via CMD. Does anyone know how to fix this? 

C:\Program Files (x86)\bulk-downloader-for-reddit-2.4.2>python "setup.py"
ERROR:root:Error parsing
Traceback (most recent call last):
  File "c:\program files (x86)\bulk-downloader-for-reddit-2.4.2\.eggs\pbr-5.7.0-py3.9.egg\pbr\core.py", line 96, in pbr
    attrs = util.cfg_to_args(path, dist.script_args)
  File "c:\program files (x86)\bulk-downloader-for-reddit-2.4.2\.eggs\pbr-5.7.0-py3.9.egg\pbr\util.py", line 272, in cfg_to_args
    pbr.hooks.setup_hook(config)
  File "c:\program files (x86)\bulk-downloader-for-reddit-2.4.2\.eggs\pbr-5.7.0-py3.9.egg\pbr\hooks\__init__.py", line 25, in setup_hook
    metadata_config.run()
  File "c:\program files (x86)\bulk-downloader-for-reddit-2.4.2\.eggs\pbr-5.7.0-py3.9.egg\pbr\hooks\base.py", line 27, in run
    self.hook()
  File "c:\program files (x86)\bulk-downloader-for-reddit-2.4.2\.eggs\pbr-5.7.0-py3.9.egg\pbr\hooks\metadata.py", line 25, in hook
    self.config['version'] = packaging.get_version(
  File "c:\program files (x86)\bulk-downloader-for-reddit-2.4.2\.eggs\pbr-5.7.0-py3.9.egg\pbr\packaging.py", line 874, in get_version
    raise Exception("Versioning for this project requires either an sdist"
Exception: Versioning for this project requires either an sdist tarball, or access to an upstream git repository. It's also possible that there is a mismatch between the package name in setup.cfg and the argument given to pbr.version.VersionInfo. Project name bdfr was given, but was not able to be found.
error in setup command: Error parsing C:\Program Files (x86)\bulk-downloader-for-reddit-2.4.2\setup.cfg: Exception: Versioning for this project requires either an sdist tarball, or access to an upstream git repository. It's also possible that there is a mismatch between the package name in setup.cfg and the argument given to pbr.version.VersionInfo. Project name bdfr was given, but was not able to be found.

C:\Program Files (x86)\bulk-downloader-for-reddit-2.4.2>

**Comment by Serene-Arc** [comment]
Try `python3 -m pip install -U wheel` first and then try installing again.

**Reply by mcsteel88** [reply]
cheers, i have already installed wheel after reading a previous post but am still getting this error.

**Reply by Serene-Arc** [reply]
How are you trying to install this? Through pip? Are you using the same install as the one with wheel?

**Reply by mcsteel88** [reply]
ive just tried to run the setup using command python "setup.py". please note i am not a developer so this stuff is a little new to me. last time i isntalled this software it was much simpler, id be happy to pay someone a small fee to show me how to set it up.

**Reply by Serene-Arc** [reply]
Ah, okay, no that's not the right way to install it. Do you want the absolute latest commits that we have **and** know git? Or do you just want to have the BDFR and have it work?

**Reply by mcsteel88** [reply]
I really just want the BDFR to work at this time and I need it ASAP. What do you mean by" latest commits that we have and know git?"

**Reply by Serene-Arc** [reply]
Nevermind then. Delete whatever you've downloaded from GitHub and run `python3 -m pip install -U bdfr`
